### ISSUE TITLE
HA 2024.6.0: Detected blocking call to open inside the event loop

### ISSUE THREAD

**Issue by noxhirsch** [Sian-Lee-SA/Home-Assistant-Switch-Manager#229]
Hi,
in Home Assistant 2024.6.0 (starting with b0) there's a lot of warnings about deteced blocking calls to open inside the event loop. Since it's just a warning and everything seems to work fine, I assume that this is just about performance - so not a high priority issue.

```
2024-05-30 10:58:32.606 WARNING (MainThread) [homeassistant.util.loop] Detected blocking call to open inside the event loop by custom integration 'switch_manager' at custom_components/switch_manager/helpers.py, line 47: data = load_yaml(f) (offender: /usr/src/homeassistant/homeassistant/util/yaml/loader.py, line 226: with open(fname, encoding="utf-8") as conf_file:), please create a bug report at https://github.com/Sian-Lee-SA/Home-Assistant-Switch-Manager/issues
```

**Comment by Stooovie** [comment]
SM doesn't open at all for me on 2024.6.0 release version. I get blank page.

**Comment by Sian-Lee-SA** [comment]
Addressed with latest release 1.3.4
